### DISCUSSION TITLE
End of life support for microsoft.extensions.configuration v7.0.0

### DISCUSSION THREAD

**Discussion by pneumowave-finlay-mcsporran** [dotnet/runtime#107975] [Q&A]
Hi,

We are in the process of applying for FDA certification and need to include information about the end-of-life dates for packages. Could someone please inform me if there are any available details regarding the end of support for microsoft.extensions.configuration version 7.0.0?

**Comment by jkotas** [comment]
This package is part of .NET 7 and it follows its lifecycle. The support for this package ended on May 14, 2024:

https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core
